nrclient/proxy_test.go

```diff
@@ -0,0 +1,113 @@
+package nrclient
+
+import (
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+)
+
+var _ = Describe("HTTP Proxy", func() {
+
+	// This lets the matching library (gomega) be able to notify the testing framework (ginkgo)
+	gomega.RegisterFailHandler(ginkgo.Fail)
+
+	// TODO Reenable
```

Comment:
Will this be done in this PR or in a later PR?

Reply:
Ah! I forgot to re-enable it after the refactor. Indeed, this needs to be reenabled in this PR. I'll fix it, thanks.